Bio/PDB/ResidueDepth.py

```diff
@@ -297,15 +295,15 @@ def _get_atom_radius(atom, rtype="united"):
         return _atomic_radii[8][typekey]
     # Co-factors, and other weirdos
     elif (
-        resname in {"FS3", "FS4"}
-        and at_name.startswith("FE")
-        and at_name.endswith(("1", "2", "3", "4", "5", "6", "7"))
+            resname in {"FS3", "FS4"}
+            and at_name.startswith("FE")
+            and at_name.endswith(("1", "2", "3", "4", "5", "6", "7"))
     ):
         return _atomic_radii[21][typekey]
     elif (
-        resname in {"FS3", "FS4"}
-        and at_name.startswith("S")
-        and at_name.endswith(("1", "2", "3", "4", "5", "6", "7"))
+            resname in {"FS3", "FS4"}
+            and at_name.startswith("S")
+            and at_name.endswith(("1", "2", "3", "4", "5", "6", "7"))
```

Comment:
I suspect these indentation change and the one above will break black formatting rules?

Reply:
This should now be fixed in [this commit](https://github.com/amorehead/biopython/commit/b22226e5225866cc8a87b18d21da5cea5987e3d8).

Reply:
That fixed the blank line change, but not this indentation.